snd/helvum/build.sh

```diff
@@ -0,0 +1,33 @@
+pkgname=helvum
```

Comment:
here too :3 maybe not idk what do you think?

Reply:
Will do